integration_tests/suite/test_call_log_generation.py

```diff
@@ -1721,3 +1728,156 @@ def test_adhoc_conference(self):
                 ),
             ),
         )
+
+    @raw_cels(
+        '''\
+        linkedid      | uniqueid      | eventtime                | eventtype                 | channame                | cid_ani   | cid_dnid   | cid_name   | cid_num   | extra                                                                                                                                                                                                                       
```

Comment:
Can we remove `cid_any` and `cid_dnid` columns or they are used?

Reply:
I don't think it's a good idea to tailor test data around tests. Tests should be valid if the underlying implementation changes without affecting the functionality and interface. In reality, all data is consumed by the service to generate the logs, however that data is used. Integration tests models the service under test as a black box, the input should be realistic and only the output matters.

Reply:
But you already strip other columns from `cel` ...
I understand your point, IMHO it's only a matter of compromise between maintain/readability and theory
```
id          | integer                  |           | not null | nextval('cel_id_seq'::regclass)
 eventtype   | text                     |           | not null | 
 eventtime   | timestamp with time zone |           | not null | 
 userdeftype | text                     |           | not null | 
 cid_name    | text                     |           | not null | 
 cid_num     | text                     |           | not null | 
 cid_ani     | text                     |           | not null | 
 cid_rdnis   | text                     |           | not null | 
 cid_dnid    | text                     |           | not null | 
 exten       | text                     |           | not null | 
 context     | text                     |           | not null | 
 channame    | text                     |           | not null | 
 appname     | text                     |           | not null | 
 appdata     | text                     |           | not null | 
 amaflags    | integer                  |           | not null | 
 accountcode | text                     |           | not null | 
 peeraccount | text                     |           | not null | 
 uniqueid    | text                     |           | not null | 
 linkedid    | text                     |           | not null | 
 userfield   | text                     |           | not null | 
 peer        | text                     |           | not null | 
 extra       | text                     |           |          | 
 call_log_id | integer                  |           |          | 
```

Reply:
I agree it's a tradeoff, and since I already agree to go in that direction I'll go "all the way" in this case, but it's a tradeoff that is bound to bite us back. The approach of colocating data with test code has benefits for consistency/understandability of tests, but the obvious cost is readability. If we wanted to optimize for readability perhaps this isn't the right strategy and we would be better off keeping sample data separate.